examples/tests/client-tests.js

```diff
@@ -0,0 +1,182 @@
+/* global QUnit */
+
+'use strict';
+
+const BASE_MAIN_CHILDREN_NUMBER = 3;
+
+QUnit.module('Pictures');
+
+QUnit.test(
+  'putPicturesInPage',
+  (assert) => {
+    assert.ok(
+      typeof window.forTesting.putPicturesInPage === 'function',
+      'Create a function `putPicturesInPage` receiving an array of pictures. The function will add each picture and its attributes to a different section with the class `picture`.',
```

Comment:
unnecessarily descriptive, I'd suggest the following:

`putPicturesInPage` takes an array of picture objects and puts them as sections in `<main>`.